incapsula/resource_subaccount.go

```diff
@@ -84,6 +92,11 @@ func resourceSubAccountCreate(d *schema.ResourceData, m interface{}) error {
 	log.Printf("[DEBUG] Account id for new sub account : %d", SubAccountAddResponse.SubAccount.SubAccountID)
 	log.Printf("[INFO] Created Incapsula subaccount %s\n", subAccountName)
 
+	err = updateDefaultDataStorageRegion(client, d)
```

Comment:
please add update method as well

Reply:
Theres are API to update account, but not sub account
https://docs.imperva.com/bundle/cloud-application-security/page/api/accounts-api.htm

Reply:
Handled